mods/ra/maps/allies-11a/allies11a-AI.lua

```diff
@@ -0,0 +1,579 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+blk1Units = 
```

Comment:
Style nit: Non `local` references should be capitalized.

Reply:
Non local references is capitalized